nix/nix-python

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+SCRIPTPATH="$(realpath "$(dirname "$0")"/.. )"
```

Comment:
I would think that we need to escape some of the `"` here (or use `'` instead). Does something like
```suggestion
SCRIPTPATH="$(dirname \"$0\")"
FLAKEPATH="$(realpath \"$SCRIPTPATH/..\")"
```
work, or am I messing up bash here?

Reply:
I think it works because its inside a subshell `$(...)` (shellcheck+my sh likes it).
+1 for having two lines so that we don't have to think about quoting

Reply:
To quote the good old POSIX standard, as to why the initial code does not need escaping the `"`: 

> The input characters within the quoted string that are also enclosed between "$(" and the matching ')' shall not be affected by the double-quotes, but rather shall define that command whose output replaces the "$(...)" when the word is expanded

(https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html, 2.2.3, paragraph 3)

Reply:
> To quote the good old POSIX standard

power move, good one! :D

Reply:
What @Kakadus didn't mention is the two of us spending 10 minutes on understanding the wording of the standard :D